content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -203,16 +203,9 @@ The default value for `default-not-ready-toleration-seconds` and `default-unreac
 
 ### DenyServiceExternalIPs
 
-This admission controller rejects all net-new usage of the `Service` field `externalIPs`.  This
-feature is very powerful (allows network traffic interception) and not well
-controlled by policy.  When enabled, users of the cluster may not create new
-Services which use `externalIPs` and may not add new values to `externalIPs` on
-existing `Service` objects.  Existing uses of `externalIPs` are not affected,
-and users may remove values from `externalIPs` on existing `Service` objects.
-
-Most users do not need this feature at all, and cluster admins should consider disabling it.
-Clusters that do need to use this feature should consider using some custom policy to manage usage
-of it.
+This admission controller rejects all net-new usage of the `Service` field `externalIPs`. The ability to set the `externalIPs` field for a `Service` is very powerful (allows network traffic interception) and not well controlled by policy. When the `DenyServiceExternalIPs` admission controller is enabled, users of the cluster may not create new `Services `which use `externalIPs` and may not add new values to `externalIPs` on existing `Service` objects. Existing uses of `externalIPs` are not affected, and users may remove values from `externalIPs` on existing `Service` objects.
+
+Most users do not need the ability to set the `externalIPs` field for a `Service` at all, and cluster admins should consider disabling this functionality by enabling the `DenyServiceExternalIPs` admission controller. Clusters that do need to allow this functionality should consider using some custom policy to manage its usage.
```

Comment:
```suggestion
Most users do not need the ability to set the `externalIPs` field for a `Service` at all, and cluster admins should consider disabling this functionality by enabling the `DenyServiceExternalIPs` admission controller. Clusters that do need to allow this functionality should consider using some custom policy to manage its usage.
```
Rather than overwriting the new words mentioned above, we can let this paragraph as it was before.

Reply:
Hi @ NitishKumar06, in my opinion leaving the paragraph as is keeps it ambiguous, which was the reason I initially opened an issue. When you consider the unchanged paragraph in the context of the sentence that follows: 

> Most users do not need this feature at all, and cluster admins should consider disabling it. Clusters that do need to use this feature should consider using some custom policy to manage usage of it.

> This admission controller is disabled by default.

Saying "this feature" should be disabled (without defining "this feature" again) and then following it with "this admission controller is disabled by default" could be misinterpreted as saying that _the feature_ is disabled by default (in fact, I had a conversation yesterday with a co-worker her misinterpreted the current wording). In my mind, the documentation should be as clear as possible, even if it might be a bit repetitive.

Reply:
Sounds good to me! Thanks